content/role-based-permissions copy.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Role based Permissions"
+metaTitle: "sensenet - Role based Permissions"
+metaDescription: "sensenet Role based Permissions"
+---
+
+sensenet offers you a sophisticated permission system, giving you the possibility to apply permission rules according to the logic you prefer.
+
+# Advantages
+Within an organization roles are created for various job functions. Using role-based permissions, you can assign users to roles taking advantage of predefined permission settings to that role. Since users are not assigned permissions directly, but only acquire them through their role (or roles), management of individual user rights becomes a matter of simply assigning appropriate roles to the user’s account; this simplifies common operations, such as adding a user, or changing a user’s department.
```

Comment:
```suggestion
Within an organization roles are created for various job functions. Using role-based permissions, you can assign users to roles taking advantage of predefined permission settings to that role. Since permissions are not directly assigned to users, but only acquire them through their role (or roles), management of individual user rights becomes a matter of simply assigning appropriate roles to specific users; this simplifies common operations, such as adding a new user, or changing a user’s department.
```